modules/apps/app-builder/app-builder-rest-impl/rest-openapi.yaml

```diff
@@ -37,6 +37,15 @@ components:
                     format: int64
                     type: integer
             type: object
+        AppDeployment:
+            properties:
+                deploymentType:
```

Comment:
should be `type` here as well, the entity is already called Deployment.

Reply:
No need for enum since a developer can define a custom deployment type. Changed to type.